cunumeric/array.py

```diff
@@ -3486,13 +3481,14 @@ def _perform_unary_reduction(
         src,
         axis=None,
         dtype=None,
-        dst=None,
+        out=None,
         keepdims=False,
         args=None,
-        check_types=True,
         initial=None,
         where=True,
     ):
+        where = convert_to_predicate_ndarray(where)
```

Comment:
I assume this is only necessary because of the fallback from `ufunc.py`; all uses of this function in `array.py` and `module.py` are internal, and should have already converted their arguments.

In that case, can you use `@add_boilerplate` instead? That way we also handle `out` automatically.

Reply:
0983a85268d3ec45a7907e04bd342e4c87611909